front/components/assistant/conversation/AssistantEditionMenu.tsx

```diff
@@ -19,7 +18,6 @@ import { assertNever, isBuilder } from "@dust-tt/types";
 import { useContext, useState } from "react";
```

Comment:
OOC but why do we have `AssistantEditionMenu.tsx` under `front/components/assistants/conversation/` and not `front/components/assistants/`?

Reply:
🤷

Reply:
Fair point, move outside of the conversation folder.